casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -57,29 +66,6 @@ class HashSetCasperTest extends FlatSpec with Matchers {
     node.tearDown()
   }
 
-  it should "not allow multiple threads to propose a block at the same time" in {
```

Comment:
Where did this test go?

Reply:
It now is redundant with the added test

Reply:
If it's redundant, did you remove the lock as well?

Reply:
As we discussed in person, I have removed the lower level lock indeed